tests/simple.sh

```diff
@@ -0,0 +1,87 @@
+#/bin/bash
+# A quick port of the travis tests to bash
+# TODO evaluate using a framework..
+# We need to be running as root..
+
+PYTHON=${1:-/usr/bin/python3}
+SOSPATH=${2:-./sosreport}
+
+NUMOFFAILURES=0
+
+run_expecting_sucess () {
+FAIL=false
+#Make sure clean
+rm stderr stdout
+
+echo "######### RUNNING $1 #########"
+$PYTHON $SOSPATH $1 2> stderr 1> stdout
+
+if [ $? -eq 0 ]; then
+  echo "### Success"
+else
+  echo "!!! FAILED !!!"
+  FAIL=true
+fi
+
+if [ -s stderr ]; then
+   FAIL=true
+   echo "!!! FAILED !!!"
+   echo "### start stderr"
+   cat stderr
+   echo "### end stderr"
+fi
+
+echo "### start stdout"
+cat stdout
+echo "### end stdout"
+
+echo "######### DONE WITH $1 #########"
+
+if $FAIL; then 
+  return 1
+else
+  return 0
+fi
+
+}
```

Comment:
It would be great to have indentation of the commands in the `run_expecting_success()` method - it is bit confusing to understand what is the method's code and what is the script's main body.